site/docs/v1/tech/getting-started/5-minute-sandbox.adoc

```diff
@@ -0,0 +1,91 @@
+---
+layout: doc
```

Comment:
I only tested the sandbox install and everything worked perfectly with the node + typescript client. Please let me know if you would like me to try out the other options as well.

Reply:
Thanks, it's all shared code, so if it works for sandbox, works for the others.